docs/php/how-to-develop-an-activity-definition-in-php.md

```diff
@@ -0,0 +1,56 @@
+---
+id: how-to-develop-an-activity-definition-in-php
+title: Develop an Activity Definition in PHP
+sidebar_label: Activity Definition
+---
+
+An Activity is a manifestation of a particular Task in the business logic.
```

Comment:
My preference would be to drop this explanation bit.
Let the context bits do this.